index.html

```diff
@@ -229,5 +229,32 @@ <h2 class="categorie-title">
                 </li>
             </ul>
         </section>
+        <section class="form" id="form-id">
+            <h1 class="form-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+            <form action="https://formspree.io/f/xvongony" method="POST" class="form-input">
```

Comment:
- [OPTIONAL] Using flex in laying out form elements is a core requirement of this milestone. you are using flex in individual form child elements like input-name and input-email, but you are not using flex for laying out these children elements of the form section. I would recommend to use flex for this purpose.